release_notes/ocp-4-12-release-notes.adoc

```diff
@@ -111,6 +111,13 @@ For more information, see xref:../networking/ingress-controller-dnsmgt.adoc#ingr
 
 For more information, see xref:../networking/hardware_networks/about-sriov.adoc#supported-devices_about-sriov[Supported devices].
 
+[id="ocp-4-12-ipv6-gratuitous-ARPs-default-SR-IOV"]
+==== IPv6 unsolicited neighbor advertisements and IPv4 gratuitous address resolution protocol now default on the SR-IOV CNI plugin
+
+Pods created with the Single Root I/O Virtualization (SR-IOV) CNI plugin (where the IP address management CNI plug-in has assigned IPs) now send IPv6 unsolicited neighbor advertisements and/or IPv4 gratuitous address resolution protocol by default onto the network. This enhancement notifies hosts of the new pod's MAC address for a particular IP to refresh ARP/Neighbor caches with the correct information.
```

Comment:
```suggestion
Pods created with the Single Root I/O Virtualization (SR-IOV) CNI plugin (where the IP address management has assigned IPs) now send IPv6 unsolicited neighbor advertisements and/or IPv4 gratuitous address resolution protocol by default onto the network. This enhancement notifies hosts of the new pod's MAC address for a particular IP to refresh ARP/Neighbor caches with the correct information.
```

Reply:
I was trying to suggest a way to avoid the repetitive phrasing of CNI plugin. Not sure if my suggestion changes your meaning or if CNI plugin is implied in the parenthetical? Hope this helps.

Reply:
@JoeAldinger It's a good note. But my understanding is that those are actually two different types of CNI plugins at work and one has to operate properly before the other can.